Monika After Story/game/zz_dump.rpy

```diff
@@ -306,9 +306,19 @@ init 999 python:
             _var_data_file.write(config.version + "\n\n")
 
             # add data lines here
-            _var_data_file.write("CUPS OF COFFEE DRANK: {0}".format(
-                persistent._mas_coffee_cups_drank
-            ))
+            #Consumables stuff
+            for consumable_id in persistent._mas_consumable_map.keys():
+                consumable = mas_getConsumable(consumable_id)
+
+                #Need to account for consumables which were removed
+                if consumable:
+                    _var_data_file.write(
+                        "{0}S OF {1} HAD: {2}\n".format(
```

Comment:
The original log message was a shoutout to certain messages in Transistor, so if we could have this vary between food and drink (`EATEN`/`DRANK`), that would be a nice touch.